types/sharedb/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/sharedb",
-    "version": "3.3.9999",
+    "version": "5.1.9999",
```

Comment:
Oh, interesting, I didn't realize we should've been updating this version in sync with new major versions of sharedb, even though the new major versions have just been for Node version updates. But syncing major+minor does seem to be the current recommendation:
https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library